howso/client/base.py

```diff
@@ -52,10 +47,17 @@ def get_version(self):
 
     @abstractmethod
     def create_trainee(
-        self, trainee, *,
+        self,
+        name=None,
+        features=None,
+        *,
+        overwrite_trainee=False,
```

Comment:
Lets make all the kwarg-only arguments sorted alphabetically.

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60